pod/pod-api-public.yaml

```diff
@@ -6738,4 +6802,235 @@ definitions:
     properties:
       languages:
         $ref: "#/definitions/StringList"
-
+  ThreadMessagesDataqueryResult:
+    description: ''
+    type: object
+    properties:
+      userId:
+        type: integer
+      threadId:
+        type: string
+      envelopes:
+        type: array
+        items:
+          $ref: '#/definitions/Envelope'
+      from:
+        type: integer
+      to:
+        type: integer
+      limit:
+        type: integer
+  Envelope:
+    description: ''
+    type: object
+    properties:
+      message:
+        type: object
+        $ref: '#/definitions/Message'
+      suppressed:
+        type: boolean
+      isShared:
+        type: boolean
+      shareCount:
+        type: integer
+      isRead:
+        type: boolean
+      isLiked:
+        type: boolean
+      likeCount:
+        type: integer
+      isBookmarked:
+        type: boolean
+      isOwnedByLocalPod:
+        type: boolean
+  Message:
+    description:
+    type: object
+    properties:
+      version:
+        type: string
+      traceId:
+        type: string
+      sid:
```

Comment:
@symphonydarlys , this is the new `sid` field included in the message detail returned by `dataquery`.